Invoking command in '/var/home/core/ros2_ws/src/ft232h': DEBUGINFOD_URLS=https://debuginfod.fedoraproject.org/ PYTHONPATH=/var/home/core/ros2_ws/build/ft232h/prefix_override:/var/home/core/ros2_ws/install/ft232h/lib/python3.12/site-packages:${PYTHONPATH} _=/usr/bin/env /usr/bin/python3 setup.py egg_info --egg-base ../../build/ft232h build --build-base /var/home/core/ros2_ws/build/ft232h/build install --record /var/home/core/ros2_ws/build/ft232h/install.log --single-version-externally-managed
Invoked command in '/var/home/core/ros2_ws/src/ft232h' returned '0': DEBUGINFOD_URLS=https://debuginfod.fedoraproject.org/ PYTHONPATH=/var/home/core/ros2_ws/build/ft232h/prefix_override:/var/home/core/ros2_ws/install/ft232h/lib/python3.12/site-packages:${PYTHONPATH} _=/usr/bin/env /usr/bin/python3 setup.py egg_info --egg-base ../../build/ft232h build --build-base /var/home/core/ros2_ws/build/ft232h/build install --record /var/home/core/ros2_ws/build/ft232h/install.log --single-version-externally-managed
